README.md

```diff
@@ -1,28 +1,29 @@
 # GDD-backend
-repository to host backend functionality for the GDD apps
+Repository to host backend functionality for the GDD apps.
```

Comment:
This repo is only used by the webapp right now. Adding 'for the GDD apps' can be confusing as we have as well the mobile app.

Reply:
Updated.